README.md

```diff
@@ -35,32 +35,32 @@ There are some differences compared to the original Uniswap that we would like t
 4. Users do not need to set approvals before their first trade
 5. 100% native meta-tx friendly for ERC-1155 implementations with native meta-tx functionalities
 6. Front-end implementations can add arbitrary fee (in addition to the 0.5%) for tokens with native meta-transactions.
-7. Less functions than Uniswap
+7. Fewer functions than Uniswap
 
 There are pros and cons to these differences and we welcome you to discuss these by openning issues in this repository.
 
 ## Contracts
 
 * [NiftyswapExchange.sol](https://github.com/0xsequence/niftyswap/blob/master/src/contracts/exchange/NiftyswapExchange.sol): The exchange contract that handles the logic for exchanging assets for a given base token.
-* [NiftyswapFactory.sol](https://github.com/0xsequence/niftyswap/blob/master/src/contracts/exchange/NiftyswapFactory.sol): The exchange factory that allows the creation of nifyswap exchanges for the tokens of a given ERC-1155 token conract and an ERC-1155 base currency.
+* [NiftyswapFactory.sol](https://github.com/0xsequence/niftyswap/blob/master/src/contracts/exchange/NiftyswapFactory.sol): The exchange factory that allows the creation of Niftyswap exchanges for the tokens of a given ERC-1155 token contract and an ERC-1155 base currency.
 
 ## Security Review
 
-Niftyswap has been audited by two independant parties and all issues discovered were addressed. 
+Niftyswap has been audited by two independent parties and all issues discovered were addressed. 
 - [Agustín Aguilar**](https://github.com/0xsequence/niftyswap/blob/master/audits/Security_Audit_Nitfyswap_Horizon_Games_1.pdf)
```

Comment:
Note that this PDF filename has "Nitfy" misspelled, so the link was not updated.